app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MessagesController < ApplicationController
+  def random_greeting
+    @random_greeting = Message.all.sample
+    render json: @random_greeting
+  end
+end
```

Comment:
- [ ] Please move this controller to `api/v1` directory, It's a good practice to place the API controllers in the API Directory. Also, make sure you fix the controller calls in the routes file after fixing this issue.

Reply:
@TedLivist Thank you, will fix this.